relentless/simulate/hoomd.py

```diff
@@ -225,11 +234,10 @@ def make_snapshot(self, sim):
             N += sim.ensemble.N[t]
 
         # cast simulation box in HOOMD parameters
-        Lx,Ly,Lz,xy,xz,yz = self.extract_box_params(sim)
-
+        Lx,Ly,Lz,xy,xz,yz,dim = self.extract_box_params(sim)
```

Comment:
Perfect, yes, this is how this should be used. An alternative you could consider is to turn the box sizes into a tuple so they can still be used that way:
```
(Lx,Ly,Lz,xy,xz,yz), dim = self.extract_box_params(sim)
box_size, dim = self.extract_box_params(sim)
```
To do that, you just add the tuple parentheses in the `return` of the function above. However, I am perfectly fine with this design too.